### ISSUE TITLE
Compatibility with Nextcloud 15

### ISSUE THREAD

**Issue by MetropolitanInfluence** [Yetangitu/owncloud-apps#112]
Hi there,
files_opds 0.8.8 seems to be working fine on NC 15.
I just needed to rise the 'max-version' value up to '15.0' in info.xml file to be able to activate the app.

**Comment by chopnico** [comment]
Any issues with files_readers?

**Comment by merrx** [comment]
I followed the link in https://apps.nextcloud.com/apps/files_opds
and used @thomas2403's fork to install the files_opds on my nextcloud 15 instance.
I had to increase the max-version to 15.0 to enable the app.

I get a integrity check error for "Certificate not valid" and my library stays at 0 books even after scheduling a rescan. A look in the database table shows no entries.

I am not sure, which fork to take...

**Comment by 4oo4** [comment]
@chopnico Nope, as long as you make sure this says the version maximum as 15.0 it seems to work:

`appinfo/info.xml` :
```

    <dependencies>
        <owncloud min-version="8.2" max-version="10.0" />
        <nextcloud min-version="8.1" max-version="15.0"/>
                <database>pgsql</database>
                <database>sqlite</database>
                <database>mysql</database>
    </dependencies>

**Comment by onliniak** [comment]
It looks like 15.0.4 is diffrent. I try 15.0 and 15.0.4 and even 15.1 too

![15.0.4](https://i.imgur.com/8JarjsJ.png)
